specs/Normative-Types-Specification.rst

```diff
@@ -416,13 +417,13 @@ A field may be optional, and may be described along with a comment:
 
 where:
 
-<*NormativeType*>
+``<NormativeType>``:
```

Comment:
How should this content be formatted?  Is there a link to the current view somewhere else?

Reply:
As above, in my understanding line 419 renders as intended by the author. He reviewed the document once and of course he may have missed this, as well as the other similar case (line 363) above, but at least he did not complain.

Reply:
...I forgot to mention: unfortunately the original was on the old EPICS 4 website which has been taken offline and I cannot reproduce the rendering. I asked Greg to review, not only but also because of this.